simtools/simtel/simtel_io_histograms.py

```diff
@@ -51,14 +51,33 @@ class SimtelIOHistograms:
         calculating trigger rate for individual telescopes.
         If false, the area thrown is estimated based on the maximum distance as given in
         the simulation configuration.
+    energy_range: list
+        The energy range used in the simulation. It must be passed as a list of floats and the
```

Comment:
Same comment here. GeV in Corsika?

Reply:
As pointed out before, yes, this is correct. We are not directly dealing with CORSIKA parameters here (they are already dealt with in eventio).

Reply:
Please remove then "as in CORSIKA configuration", as this is not correct (CORSIKA is strictly in GeV).